apps/src/gamelab/GameLab.js

```diff
@@ -177,6 +216,34 @@ GameLab.prototype.init = function (config) {
   });
 };
 
+function quote(str) {
+  return '"' + str + '"';
+}
+
+/**
+ * Returns a list of options (optionally filtered by type) for code-mode
+ * asset dropdowns.
+ */
+GameLab.prototype.getAssetDropdown = function (typeFilter) {
+  var options = assetListStore.list(typeFilter).map(function (asset) {
+    return {
+      text: quote(asset.filename),
+      display: quote(asset.filename)
+    };
+  });
+  var handleChooseClick = function (callback) {
+    showAssetManager(function (filename) {
+      callback(quote(filename));
+    }, typeFilter);
+  };
+  options.push({
+    text: 'Choose...',
```

Comment:
I know we're not really localizing this yet, but can we throw this string into a loc file?